hal-roborio/setup.py

```diff
@@ -7,6 +7,11 @@
 
 from setuptools import setup
 
+# Check to see if we're on a RoboRIO before potentially downloading anything
+if __name__ == '__main__' and len(sys.argv) > 1 and sys.argv[1] == 'install':
```

Comment:
In the hal-sim file this part is at the bottom. It needs to go there for it to work

Reply:
Why does it need to go on the bottom?

Reply:
idk. it doesn't work at the top. It's like name isn't `__main__` yet.

Reply:
`__name__` doesn't just arbitrarily change midway through a script. You're doing something wrong.

Reply:
`pip install .`

Reply:
Ah. `sys.argv[1]` is not 'install'. It's 'egg_info'

Reply:
virtualenv pip did some funky stuff, and I had to remove from cache